images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -3,8 +3,7 @@ FROM oryxdevmcr.azurecr.io/private/oryx/githubrunners-buildpackdeps-${DEBIAN_FLA
 ARG DEBIAN_FLAVOR
 ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
 # Install basic build tools
-RUN LANG="C.UTF-8" \
```

Comment:
Do you see any issue with keeping this `LANG="C.UTF-8"` statement here (and in the other places that you have it removed)? I know it may be redundant to have it defined here _and_ in the `ENV` command that you've added below in these files, but I'm wondering if there was a reason that this environment variable was set here because it was needed for the duration of the `RUN` command when installing packages via `apt-get install`.

Reply:
That's the part I was kinda worried as well. I'm not sure if this is required for apt-get install. It seems like other runtime imgs doesn't have LANG env defined prior to apt-install tho. Also, it didn't have any error thrown when building the base images without the env defined there.

Reply:
just did a quick search. seems like setting LANG before apt-get is a recommended procedure for troubleshooting in Ubuntu guide. I think it'll be fine leaving `LANG="C.UTF-8"` out before the `apt-get install`